### PR TITLE
Add .gitattributes file

### PR DESCRIPTION
**Pull request by Abijeet**
As recommended here: https://www.mediawiki.org/wiki/Manual:Developing_libraries#Packagist_guidelines

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.editorconfig export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.gitreview export-ignore
```
Comment (Nikerabbit) [inline .gitattributes:4]: We don't have this file

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Doxyfile export-ignore
```
Comment (Nikerabbit) [inline .gitattributes:5]: We don't have this file

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+composer.json export-ignore
+.phpcs.xml export-ignore
+phpunit.xml.dist export-ignore
```
Comment (Nikerabbit) [inline .gitattributes:8]: We don't have this file, but we have phpunit.xml

File: .gitattributes [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.editorconfig export-ignore
```
Comment (Nikerabbit) [inline .gitattributes:1]: How about these:
.github/workflows
.eslintrc.json
composer.lock (not sure about this)
package-lock.json
package.json
phpunit.xml

Unsure:
docs, CHANGELOG.md, COPYING, README.md (not needed at runtime, but maybe useful to keep?)

Comment (Abijeet) [reply .gitattributes:1]: Added,

```
.github
.eslintrc.json
package-lock.json
package.json
phpunit.xml
docs/
```
Left the other files in.

**Pull request merged**